.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+# Describe the bug #
+
+A clear and concise description of what the bug is.
+
+# Steps to reproduce #
+
+Steps to reproduce the behavior:
+
+1. Go to '...'
+1. Click on '....'
+1. Scroll down to '....'
+1. See error
+
+# Actual results #
+
+Describe what happened
+
+# Expected behavior #
+
+A clear and concise description of what you expected to happen.
+
+# Additional context #
+
+Add any other context about the problem here.
+
+For example:
+
+- Image/version of Ceph CSI driver
+- helm chart version
+- Kubernetes cluster version
+- Logs
```

Comment:
(future) When we have a troubleshooting/self-help guide we should link it here.

Reply:
Sure.